manifests/core-bases/base/deployment.yaml

```diff
@@ -84,7 +84,7 @@ spec:
             - --pass-access-token
             - '--openshift-delegate-urls={"/": {"resource": "projects", "verb": "list"}}'
             - --skip-auth-regex=^/metrics
-          image: oauth-proxy
+          image: $(oauth-proxy-image)
```

Comment:
Only concern I have is this probably should have been named "auth-image" or something more generic. OAuth Proxy is being swapped out for Kube RBAC Proxy soon.